salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountDetailsPage/LightningAccountDetailsPage.java

```diff
@@ -22,19 +22,18 @@
 public class LightningAccountDetailsPage extends BasePage implements IAccountDetailsPage {
 
     private static final int ACCOUNT_STRING_SIZE = 8;
-    private String accountInfoXpath = "//span[.='%1$s']/../../div[2]//%2$s";
-    private String opportunityToSearchXpath =
+    private static final String ACCOUNT_INFO = "//span[.='%1$s']/../../div[2]//%2$s";
+    private static final String OPPORTUNITY_TO_SEARCH =
```

Comment:
Please, keep this By and Strings locators in lowerCase to be consistent to all the project, we use Upper case only for constants in primitive data type (int, Strings, char) but not applicable for String locators, so avoid to put them as final.